packages/node_modules/@webex/test-users/src/whistler.js

```diff
@@ -0,0 +1,159 @@
+import btoa from 'btoa';
+import {request} from '@webex/http-core';
+
+/**
+ * Fetches credentials to talk to the test_users_s endpoint
```

Comment:
```suggestion
 * Fetches credentials to talk to the test users's endpoint
```

Reply:
This whistler file doesn't talk to the `test_users_s` convo endpoint.